Bixal/move.mil
# Phone number validation and nightmare move menu updates. #340 and #341

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Updates Drupal
- Verify phone number is not empty when reading the XML file
- Updates the custom_move_mil_menus module, changing the menu item associated with the Nightmare Moves page to reflect the requested name change to Moving Stories.
- Updates the item's machine name as well as the title and target path.

## Testing

To verify the changes proposed in this pull request…


1.  UPload XML file
2. Go to the locator maps tool and look for "Wright Patterson AFB"
3. You should see the results.
4. Pull code and clear cache
5. Go to /moving-guide/nightmare-moves and change both the page name and the custom url path to match the updates to the menu item (/moving-guide/moving-stories)
 6. Refresh and check the updated menu item to make sure it works as intended



## Screenshots
Phone number validation:
![image](https://user-images.githubusercontent.com/48218363/55345907-187dfb00-547f-11e9-9c0f-108e933fc566.png)

Menu updates:
Before:
![image](https://user-images.githubusercontent.com/48218363/55346087-993cf700-547f-11e9-985e-8ae8e03ccf1c.png)

After:
![image](https://user-images.githubusercontent.com/48218363/55346114-ab1e9a00-547f-11e9-8e2c-ffd22af8e3d9.png)
